src/main/java/lotto/domain/Lotto.java

```diff
@@ -11,40 +11,48 @@
 public class Lotto {
 
     private final List<LottoNumbers> lottoNumbersList;
-    private LottoResult lottoResult;
 
     public static Lotto initAllRoundLottoNumbers(LottoGenerator lottoGenerator, int tryCount) {
+        return initAllRoundLottoNumbers(new Lotto(new ArrayList<>()), lottoGenerator, tryCount);
+    }
+
+    public static Lotto initAllRoundLottoNumbers(Lotto manualLotto, LottoGenerator lottoGenerator, int tryCount) {
         List<LottoNumbers> lottoNumbersList = new ArrayList<>();
-        for (int i = 0; i < tryCount; i++) {
-            lottoNumbersList.add(new LottoNumbers(lottoGenerator.executeStrategy()));
-        }
+        insertManualLotto(manualLotto, lottoNumbersList);
+        insertRandomLotto(lottoGenerator, tryCount, lottoNumbersList);
         return new Lotto(lottoNumbersList);
     }
 
     public Lotto(List<LottoNumbers> lottoNumbersList) {
         this.lottoNumbersList = lottoNumbersList;
     }
 
-    public List<LottoNumbers> totalRoundLottoNumberList() {
-        return lottoNumbersList;
+    private static void insertRandomLotto(LottoGenerator lottoGenerator, int tryCount, List<LottoNumbers> lottoNumbersList) {
+        for (int i = 0; i < tryCount; i++) {
+            lottoNumbersList.add(new LottoNumbers(lottoGenerator.executeStrategy()));
+        }
     }
 
-    public void updateWinningRankList(LottoNumbers winningLottoNumbers, LottoNumber bonusNumber) {
-        lottoResult = new LottoResult(lottoRankList(winningLottoNumbers, bonusNumber));
+    private static void insertManualLotto(Lotto manaulLotto, List<LottoNumbers> lottoNumbersList) {
+        if (!manaulLotto.isEmpty()) {
+            lottoNumbersList.addAll(manaulLotto.totalRoundLottoNumberList());
+        }
     }
 
-    public List<LottoRank> lottoRankList(LottoNumbers winningLottoNumbers, LottoNumber bonusNumber) {
-        return lottoNumbersList.stream()
-                .map(lottoNumbers -> winningLottoNumbers.lottoRank(lottoNumbers, bonusNumber))
-                .filter(lottoRank -> lottoRank != LottoRank.NONE).collect(Collectors.toList());
+    public boolean isEmpty() {
+        return lottoNumbersList.isEmpty();
     }
 
-    public LottoResult getLottoResult() {
-        return lottoResult;
+    public List<LottoNumbers> totalRoundLottoNumberList() {
+        return lottoNumbersList;
     }
 
-    public double calculateMarginPercent(int purchaseAmount) {
-        return lottoResult.calculateMarginPercent(purchaseAmount);
+    public LottoResult winningRanks(WinningLotto winningLotto) {
+        List<LottoRank> lottoRanks = lottoNumbersList.stream()
```

Comment:
하단과 같은 방법으로도 반환 가능하겠네요!

```suggestion
        return lottoNumbersList.stream()
                .map(winningLotto::lottoRank)
                .filter(lottoRank -> lottoRank != LottoRank.NONE)
                .collect(Collectors.collectingAndThen(Collectors.toList(), LottoResult::new));
```